pytest-embedded/pytest_embedded/unity.py

```diff
@@ -206,8 +206,9 @@ class JunitMerger:
     SUB_JUNIT_FILENAME = 'dut.xml'
     # multi-dut junit reports should be dut-[INDEX].xml
 
-    def __init__(self, main_junit: Optional[str]) -> None:
+    def __init__(self, main_junit: Optional[str], unity_test_report_mode: Optional[str] = 'replace') -> None:
         self.junit_path = main_junit
+        self.unity_test_report_mode = unity_test_report_mode
```

Comment:
```suggestion
        self.unity_test_report_mode = unity_test_report_mode or 'replace'
```

besides, better to put the strings under enum

```python
class UnityTestReportMode(str, Enum):
    REPLACE = 'replace'
    ...
```

this is helpful since it's easy to make spelling mistakes while using bare strings